dotcom-rendering/src/web/components/App.tsx

```diff
@@ -127,35 +126,7 @@ export const App = ({ CAPI }: Props) => {
 		theme: pillar,
 	};
 
-	const adTargeting: AdTargeting = buildAdTargeting({
```

Comment:
`adTargeting` can be determined purely on server side. With Islands there is no need to `buildAdTargeting` on the client.

Reply:
Oh I see its built server side and is available client side - ignore!

Reply:
Sorry, perhaps my comment wasn't super clear. What I was trying to say here is that we don't need to recreate the `adTargeting` object on the client, it only needs to be created once on the server. `adTargeting` is provided to the youtube atom player (passed down as a prop when server rendered https://github.com/guardian/dotcom-rendering/blob/dlawes/youtube-block-island/dotcom-rendering/src/web/components/YoutubeBlockComponent.importable.tsx#L200) and will be available to youtube atom player on the client

Reply:
Thats great yes thanks for explaining 👍